workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-spring-boot/src/main/java/com/liferay/partner/QueueListener.java

```diff
@@ -375,6 +600,118 @@ HttpHeaders.AUTHORIZATION, _getAuthorization()
 			).block());
 	}
 
+	private void _unassignUser(
+			String accountName, String contactEmailAddress,
+			String contactRoleName, String externalReferenceCode)
+		throws Exception {
+
+		if (contactRoleName.equals("Partner Manager")) {
+			String pmRoleName = "Partner Manager (PM)";
+
+			_unassignUserFromRegularRole(contactEmailAddress, pmRoleName);
+			_unassignUserFromAccountRole(
+				externalReferenceCode, contactEmailAddress,
+				"[Account] " + pmRoleName);
+
```

Comment:
empty line here isn't necessary. It should at least be consistent with the other if block though